docs/sources/mimir/set-up/migrate/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 aliases:
-  - migration-guide/
-  - ../migrate/
+  - ../migration-guide/ # /docs/mimir/<MIMIR_VERSION>/migration-guide/
+  - ../migrate/ # /docs/mimir/<MIMIR_VERSION>/migrate/
```

Comment:
This comment is helpful as a reminder of the intended redirect destination, per @jdbaldry